.gitattributes

```diff
@@ -94,3 +94,7 @@ yarn.lock                                   linguist-generated
 **/dist/*.mjs                               linguist-generated
 **/dist/*.cjs                               linguist-generated
 **/dist/*.cjs.map                           linguist-generated
+
+## Codegen Models
+**/example/lib/models/**                          linguist-generated
+**/example/lib/models/gen2/Gen2ModelProvider.dart linguist-generated=false
```

Comment:
Q: Is this file not generated? It appears to be generated. If it is manually edited can you leave a comment in the file noting what manual changes were made?